samples/quickstart.js

```diff
@@ -19,27 +19,28 @@
 // Imports the Google Cloud client library
 const Bigtable = require('@google-cloud/bigtable');
 
-// Creates a client
+// The name of the Cloud Bigtable instance
+const instanceName = 'my-bigtable-instance';
+// The name of the Cloud Bigtable table
+const tableName = 'my-table';
+// Creates a Bigtable client
 const bigtable = new Bigtable();
 
-// The name for the new instance
-const instanceName = 'my-new-instance';
+// Connect to an existing instance:my-bigtable-instance
+const instance = bigtable.instance(instanceName);
 
-// Creates the new instance
-bigtable
-  .createInstance(instanceName, {
-    clusters: [
-      {
-        name: 'my-cluster',
-        location: 'us-central1-c',
-        nodes: 3,
-      },
-    ],
-  })
-  .then(() => {
-    console.log(`Instance ${instanceName} created.`);
-  })
-  .catch(err => {
-    console.log('ERROR:', err);
+// Connect to an existing table:my-table
+const table = instance.table(tableName);
+
+//Read data from my-table
+table.createReadStream()
```

Comment:
creatReadStream is the most complex version of reads.  Can this be switched to reading just the 'r1' row? (I think it's `let [singeRow] = await table.row('r1').get();`